modules/core/output_modules.php

```diff
@@ -498,7 +498,7 @@ protected function output() {
         if (DEBUG_MODE) {
             foreach (glob(APP_PATH.'modules/**', GLOB_ONLYDIR | GLOB_MARK) as $name) {
                 $rel_name = str_replace(APP_PATH, '', $name);
-                $mod = str_replace(array('modules/', '/'), '', $rel_name);
+                $mod = str_replace(array('modules/', '/', '\\'), '', $rel_name);
```

Comment:
Why don't you try with DIRECTORY_SEPARATOR constant when you see windows-specific behavior problems?

Reply:
@victor i am not sure if the DIRECTORY SEPARATOR is needed here
the idea is to replace "modules/" "/"  and "\" by an empty string "" so that we stay with only the name of the module as return value
On Windows the original string is comming with "\" but we didn't have "\" in the replacement

Reply:
please be careful with your @-mentions

Reply:
I see you added forward slash to the array of replacements. That's the DIRECTORY_SEPARATOR constant in windows. Please use it instead of guessing or trying to replace all possible variants.

Reply:
@kroky on windows the $rel_name has values like **modules/2fa\**. This is one of values I copied from a var_dump that I did on windows. That why I think the / and \ are necessary there.

Reply:
> @kroky i am not sure if the DIRECTORY SEPARATOR is needed here the idea is to replace "modules/" "/" and "\" by an empty string "" so that we stay with only the name of the module as return value On Windows the original string is coming with "\" but we didn't have "\" in the replacement

@ElvisAns Windows returns values like `modules/2fa\` in $rel_name variable. With the modules/ replaced we just need to remove the DIRECTORY_SEPARATOR as for this example the remaining part will be `2fa\`

Reply:
Thanks @kroky & @josaphatim

Reply:
Hmm, I thought it would be something like 'modules' . DIRECTORY_SEPARATOR, so you don't end up testing for forward and backward slashes.

Reply:
Aha!
@kroky We are back to what i said initially
The original string contain a mixture of "/" and "\" when we use DIRECTORY_SEPARATOR we will only replace "\" on windows and "/" on linux but not both at the same resulting in an issue again
![image](https://github.com/cypht-org/cypht/assets/35831811/258af823-827a-427d-95ae-f126a2847c67)

Reply:
If you end up with a mixture of these slashes, you are probably not using the directory separator correctly. I still see hard-coded forward slashes in modules/ string and in glob pattern.

Reply:
You are absolutely right @kroky 🙌